lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -126,15 +126,18 @@ class DetailsRenderer {
     const theadElem = this._dom.createChildOf(tableElem, 'thead');
     const theadTrElem = this._dom.createChildOf(theadElem, 'tr');
 
-    for (const heading of details.itemHeaders) {
-      this._dom.createChildOf(theadTrElem, 'th').appendChild(this.render(heading));
-    }
+    details.itemHeaders.forEach((heading, index) => {
+      const itemType = details.items[0][index].type;
```

Comment:
this kind of goes counter to supporting different types per item in the column :) Do we have places where we use that yet? Could cause issues depending on which item is first.

FWIW the headers (as generated by `Audit.makeV2TableDetails`) have an `itemType` property, we just drop them from the typedef by the time it gets to the renderer. Could pipe them through to here if you wanted.